website/content/docs/agent/config-entries/ingress-gateway.mdx

```diff
@@ -233,6 +274,66 @@ Listeners = [
 ]
 ```
 
+</Tab>
+<Tab heading="Kubernetes YAML">
+
+Set up a wildcard HTTP listener on an ingress gateway named "us-east-ingress" to proxy traffic to all services in the datacenter.
+Also make two services available over a custom port with user-provided hosts, and enable TLS on every listener:
+
+```yaml
+apiVersion: consul.hashicorp.com/v1alpha1
+kind: IngressGateway
+metadata:
+  name: us-east-ingress
+spec:
+  tls:
+    enabled: true
+  listeners:
+    - port: 8080
+      protocol: http
+      services:
+        - name: '*'
+    - port: 4567
+      protocol: http
+      services:
+        - name: api
+          hosts: ['foo.example.com', 'foo.example.com:4567']
+        - name: web
+          hosts: ['website.example.com', 'website.example.com:4567']
+```
+
+</Tab>
+<Tab heading="Kubernetes YAML (Consul Enterprise)">
+
+Set up a wildcard HTTP listener on an ingress gateway named "us-east-ingress" to proxy traffic to all services in the frontend namespace.
+Also make two services in the frontend namespace available over a custom port with user-provided hosts, and enable TLS on every listener:
+
+```yaml
+apiVersion: consul.hashicorp.com/v1alpha1
+kind: IngressGateway
+metadata:
+  name: us-east-ingress
+  namespace: default
+spec:
+  tls:
+    enabled: true
+  listeners:
+    - port: 8080
+      protocol: http
+      services:
+        - name: '*'
+          namespace: frontend
+    - port: 4567
+      protocol: http
+      services:
+        - name: api
+          namespace: frontend
+          hosts: ['foo.example.com', 'foo.example.com:4567']
+        - name: web
+          namespace: frontend
+          hosts: ['website.example.com', 'website.example.com:4567']
```

Comment:
Similar to the changes above
```suggestion
        - name: api
          namespace: frontend
          hosts:
            - 'foo.example.com'
            - 'foo.example.com:4567'
        - name: web
          namespace: frontend
          hosts:
            - 'website.example.com'
            - 'website.example.com:4567'
```

Reply:
I looked through the rest of the section where we use `[]` format for slices vs the `- entry` format and I think they look fine, so feel free to skip this suggestion. I still am on the fence about it, but it definitely follows the pattern we have in other places.